proposals/artifacts.md

```diff
@@ -0,0 +1,58 @@
+
+By providing an OCI artifact definition, the community can continue to innovate, focusing on new artifact types without having to build yet another storage solution (YASS). 
+
+## Proposal
+Under the http://github.com/opencontainers organization:
+
+- Create a new **artifacts** repository, named http://github.com/opencontainers/artifacts
+- Update [distribution][distribution-spec] to generically reference [manifest][image-manifest] and [index][image-index], with image as one of many artifact types it can store. 
+
+## Contents
+
+The repository will serve 2 primary goals:
+
+1. **artifact authors** - guidance for authoring new artifact types. Including a clearing house for well known artifact types.
+1. **registry operators and vendors** - guidance for how operators and vendors can support new artifact types, including how they can opt-in or out of well known artifact types. Registry operators that already implement `media-type` filtering will not have to change. The artifact repo will provide context on how new `media-type`s can be used, and how `media-type`s can be associated with a type of artifact. 
+
+### Initial Maintainers
+Initial maintainers of the artifacts project would be :
+* Steve Lasker <steve.lasker@microsoft.com> (@stevelasker)
+* Derek McGowan <derek.mcgowan@docker.com> @derekmcgowan
+* Mike Brown <brownwm@us.ibm.com> @mikebrow
+
+### Code of Conduct
+This project would incorporate (by reference) the OCI [Code of Conduct][code-of-conduct].
+
+### Governance and Releases
+This project would incorporate the Governance and Releases processes from the OCI project template: https://github.com/opencontainers/project-template.
+
+### Project Communications
+Both of the proposed projects would continue to use existing channels in use by the OCI developer community for communication including:
+* GitHub for issues and pull requests
+* The dev@opencontainers.org email list
+* The weekly OCI developer community conference call
+* The #OpenContainers IRC channel
+
+### Versioning / Roadmap
+This repository will not be versioned, but will be continuously updated with a list of versioned types with historical references. This repository will not have releases.
+
+Artifacts will reference specific [distribution][distribution-spec], [index][image-index] and [manifest][image-manifest] versions in its examples and references for capabilities.
+
+## Frequently Asked Questions (FAQ)
+
+**Q: Does this change the OCI Charter or Scope Table?**
+
+A: No.  Artifacts are a prescriptive means of storing [index][image-index] and [manifest][image-manifest] within [distribution][distribution-spec] implementations. 
```

Comment:
/s/  / /

Reply:
?

Reply:
He is referring to the double space. That replace syntax with spaces doesn't show up so well on the comments @mikebrow. There was another spot on line 8 which had an unintentional double space as well.